src/lib/repositories/impl_v2/order_status_history_repository_impl.py

```diff
@@ -1,14 +1,19 @@
 from datetime import datetime
 
-from sqlalchemy import desc
+from sqlalchemy import desc, text
 
 from src.constants.audit import Status
-from sqlalchemy.sql import func
 from src.lib.entities.sqlalchemy_orm_mapping import OrderStatusHistory
 from src.lib.repositories.order_status_history_repository import (
     OrderStatusHistoryRepository,
 )
 
+sql_query_to_update_etl_status = """
+          UPDATE order_status_histories
+            SET etl_status = 'PROCESSED'
+            WHERE  mongo_order_status_history_uuid in order_status_history_uuids  
```

Comment:
Don't you need to send an array of order_status_history ids instead?

When you migrate from postgres to mongo:

1. You load the data from prostgres that does not exist in mongo
2. You transform that data to a mongo document
3. You load this data into mongo
4. Create relationship between the document you added and the postgres order status history id with the inserted records.
5. Update the array of postgres order status history ids you used, by setting processed status and the mongo order status history id to each corresponding record

Reply:
done.